docs/cluster-configuration.md

```diff
@@ -160,6 +160,10 @@ reverted back to false.
 
 **Default**: `false`
 
+### deployKubeSecondaryDNS Feature Gate
+Set the `deployKubeSecondaryDNS` feature gate to true to allow deploying KubeSecondaryDNS by CNAO.
```

Comment:
maybe add an hyperlink here to https://github.com/kubevirt/kubesecondarydns

Reply:
Done